board/raspberrypi/rpi/rpi.c

```diff
@@ -482,7 +482,7 @@ int ft_board_setup(void *blob, bd_t *bd)
 	 * should be more intelligent, and e.g. only do this if no enabled DT
 	 * node exists for the "real" graphics driver.
 	 */
-	lcd_dt_simplefb_add_node(blob);
+	//lcd_dt_simplefb_add_node(blob);
```

Comment:
Just remove this line instead of commenting it.

Reply:
Done